### PR TITLE
New DuckDNS service support and little fixes

### PR DESCRIPTION
**Pull request by Tungsteno74**
## What(What have you changed?)
1. - [x] Add support to the [DuckDNS](https://www.duckdns.org/)  service
2. - [x] Fix Rackspace and Aurora dependencies exclusion when not used
3. - [x] Add MockResponse.text in test_utils.py as fake object response attribute by request.get or request.post for testing tool
4. - [x] Fix encoding of README.md
5. - [x] Add DuckDNS updates in Readme.md

## Why(Why did you change it?)

1. I'd like to import this package to  [Home Assistant](https://www.home-assistant.io/) project throught PyPi to use the DuckDNS service.
2. Rackspace and Aurora dependencies variable was not setted up correctly when not choosen.
3. Since DuckDNS service's reply is a basic html text page (`OK` done or `KO` fail) we need to use the `text` attribute of Requests response that was not implemented in the MockResponse class to work properly in the test suite.
4. When install sewer directly from git with the following command:
_python -m pip install git+https://github.com/komuw/sewer.git_
i receive an:
`UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 804: ordinal not in range(128)`
that refers to the string:
`Let’s Encrypt is a free, automated, and open Certificate Authority. - https://letsencrypt.org`
the problem here is the fake accent of Let’s

Hello @komuw,
these are fixes for #116

Since i like to import this package on Home Assistant (and for to that it must be on PyPi), please could you update these fixes in the next sewer release? 

Thanks

### CODE REVIEW

File: sewer/dns_providers/auroradns.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     from libcloud.dns.types import Provider, RecordType
     import tldextract
 except ImportError:
-    aurora_dependencies = True
+    aurora_dependencies = False
```
Comment (komuw) [inline sewer/dns_providers/auroradns.py]: thanks. 
I do not know how I missed this

**Comment by komuw** [comment]
@Tungsteno74 
Thanks for the PR. 

I have just released v0.7.1 `https://pypi.org/project/sewer/0.7.1/` . Try it out

File: sewer/dns_providers/auroradns.py (hunk 1 above)
Comment (Tungsteno74) [reply sewer/dns_providers/auroradns.py]: Thanks to you for your package :)

**Comment by Tungsteno74** [comment]
Sorry I forgot to add me to contributors.
Could you add me to the file? 
Thanks

**Comment by komuw** [comment]
Done: https://github.com/komuw/sewer/commit/14095fcdf86d0171d67e012e171978eb78fe4de9

**Pull request merged**